features/cache.feature

```diff
@@ -30,6 +44,27 @@ Feature: Manage oEmbed cache.
       Success: Cleared oEmbed cache.
       """
 
+  Scenario: Clear all oEmbed caches
+    # No oEmbed caches yet
+    When I try `wp embed cache clear --all`
+    Then STDERR should be:
+      """
+      Error: No oEmbed caches to clear!
+      """
+
+    # Generate some various oEmbed caches types
+    When I run `wp post generate --post_type=oembed_cache --post_date="2000-01-01" --post_title=$(wp eval 'echo md5( "foo" );') --count=10`
+    And I run `wp post meta add 1 _oembed_$(wp eval 'echo md5( "foo" );') foo`
+    And I run `wp post meta add 1 _oembed_$(wp eval 'echo md5( "bar" );') bar`
```

Comment:
Can we use WordPress APIs to generate real oEmbed cache values?

If we do so, these will be true integration tests and can alert us to breakage if core ever changes its behavior.